Formula/secp256k1.rb

```diff
@@ -6,17 +6,31 @@ class Secp256k1 < Formula
   version "0.0.0.54" # Fake version number to make updates easier.
   head "https://github.com/bitcoin-core/secp256k1.git"
 
+  option "with-enable-module-recovery", "Enable ECDSA pubkey recovery module"
+
   depends_on "automake" => :build
   depends_on "autoconf" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "gmp"
 
   def install
+    args = %W[
+      prefix=#{prefix}
+      --disable-silent-rules
+    ]
+    # Workaround for https://github.com/bitcoin-core/secp256k1/issues/674
```

Comment:
Do you know if this is definitely still an issue? I tested after upgrading to Xcode 11.2, as one of the posters in that thread also did, and it seems to work without this block now.

Reply:
Fixed